docker/developers/s3/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.*
```

Comment:
Based on the .gitignore / folder structure, I was expecting files to end up in the directories for a given emulated bucket...is that incorrect? I see things working as expected when I use the Minio UI to navigate to a given bucket, but the files aren't actually in the folders.

Reply:
(I should clarify that I do see mp3/json files being added the `cdo_sound_library` and `cdo_restricted` directories, just not files or sources like `main.json`)

Reply:
The files that make up the local S3 buckets are in docker volumes. Take a look at the `minio` service description. The files in the paths are sourced from somewhere else, so have to be downloaded / generated into those directories, and then uploaded to our emulated "s3". The files in the directories are a cache of sorts... so if they exist already and you destroy your local S3 buckets, they will quickly populate again from them.

Reply:
Gotcha! Another q then -- why do the `cdo_v3_files` and `cdo_v3_sources` directories exist then? I see they get used when deciding which minio S3 buckets to create in `install-s3.sh`, but do they have another purpose (eventually house `populate.rb` file(s))?

Reply:
Yeah, it tells that script to make that bucket, and then path would contain a `README.md` to explain to devs what that bucket is for, what limitations exist in the dev environment, etc, I'd think. And any configuration / populators that make the bucket work.